### PR TITLE
Refactoring storage variables, imports, renaming

### PR DESCRIPTION
**Pull request by achab**
This PR aims at refactoring several parts. Each commit contains a given part:
1. name storage variable using our conventation: `NamespaceName_storage_variable_name`
2. removed unused imports thanks to amarna
3. renamed constants to follow snake_case, and renamed "token" to "asset" for clarity
4. renamed felt related constants using more readable names:
    - `P` to `CAIRO_FIELD_ORDER`
    - `P-1` to `MAX_UNSIGNED_FELT`
    - `half_P` to `MAX_SIGNED_FELT`
    - `signed_MIN` to `MIN_SIGNED_FELT`

### CODE REVIEW

**Comment by EdgarBarrantes** [comment]
You're having the OZ contracts change in error message failure. Though there's another one [here](https://github.com/aave-starknet-project/aave-starknet-core/runs/8155737538?check_suite_focus=true#step:6:164) that I'm not sure what's related to.

**Review by hsouf — APPROVED** [review]
Thank you @achab for the refactoring! let's ship it

File: .github/workflows/cairo-format.yml [hunk 1]
```diff
@@ -6,17 +6,20 @@ on:
...
     strategy:
       matrix:
-        python-version: ["3.7"]
+        python-version: ["3.7"] # Cairo was tested with this version
     steps:
       - uses: actions/checkout@v3
 
       - name: install cairo
-        run: pip install cairo-lang
+        run: pip install cairo-lang==$CAIRO_VERSION
```
Comment (EdgarBarrantes) [inline .github/workflows/cairo-format.yml:22]: Ty for this!

**Review by EdgarBarrantes — APPROVED** [review]
Amazing work! Thanks for making our codebase cleaner. :shipit:

**Pull request merged**